### PR TITLE
Use pipx to install DonPAPI after they moved to pyproject.toml

### PR DESCRIPTION
**Pull request by spameier**
(reopening of #183, because I deleted the branch)

# Description
DonPAPI is now a proper Python package using `pyproject.toml`. Therefore the installation through `requirements.txt` fails.

### CODE REVIEW

File: sources/install/package_ad.sh [hunk 1]
```diff
@@ -660,13 +660,10 @@ function install_pywsus() {
...
-    ./venv/bin/python3 -m pip install -r ./requirements.txt
-    add-aliases donpapi
-    add-history donpapi
-    add-test-command "DonPAPI.py --help"
+    fapt swig
+    python3 -m pipx install git+https://github.com/login-securite/DonPAPI
+    add-history DonPAPI
+    add-test-command "DonPAPI --help"
     add-to-list "donpapi,https://github.com/login-securite/DonPAPI,Python network and web application scanner"
 }
```
Comment (QU35T-code) [inline sources/install/package_ad.sh:669]: You need to delete the alias file : https://github.com/ThePorgs/Exegol-images/blob/dev/sources/assets/zsh/aliases.d/donpapi

**Review by QU35T-code — CHANGES_REQUESTED** [review]

File: sources/install/package_ad.sh (hunk 1 above)
Comment (spameier) [reply sources/install/package_ad.sh:669]: Done that, thanks :)

**Comment by spameier** [comment]
I just noticed that the description `Python network and web application scanner` is also wrong. Should I change that?

File: sources/install/package_ad.sh [hunk 2]
```diff
@@ -660,13 +660,10 @@ function install_pywsus() {
...
-    python3 -m venv ./venv/
-    ./venv/bin/python3 -m pip install -r ./requirements.txt
-    add-aliases donpapi
-    add-history donpapi
-    add-test-command "DonPAPI.py --help"
+    fapt swig
+    python3 -m pipx install git+https://github.com/login-securite/DonPAPI
+    add-history DonPAPI
+    add-test-command "DonPAPI --help"
     add-to-list "donpapi,https://github.com/login-securite/DonPAPI,Python network and web application scanner"
```
Comment (Dramelac) [inline sources/install/package_ad.sh:667]: Yes you can update the description, thank you!

**Review by Dramelac — CHANGES_REQUESTED** [review]
You also need to update the history of DonPAPI (`sources/assets/zsh/history.d/donpapi`) because the command name changed from `DonPAPI.py` to `DonPAPI`.
You can add other command history of donPAPI if you want but its not mandatory.

**Comment by Dramelac** [comment]
(and no need to force push, you can add commit so we can follow your changes)

**Review by Dramelac — APPROVED** [review]

**Review by QU35T-code — APPROVED** [review]

**Comment by Dramelac** [comment]
Will be merged as soon as the workflow end, thank you for the PR @spameier !

**Pull request merged**